### ISSUE TITLE
how does this repo use node.js `events` in a front-end react app?

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet-editor#175]
How does this line work?
https://github.com/3DStreet/3dstreet-editor/blob/master/src/lib/Events.js#LL1C9-L1C9

ChatGPT has the answer:

In the provided devDependencies, Webpack (version 5.73.0) is responsible for bundling the 'events' module and including the necessary polyfill for it to work in a browser environment.

Starting with Webpack 5, many of the core Node.js polyfills have been removed by default, and developers are expected to use the specific packages from npm that provide the same functionality for the browser. However, the 'events' module is one of the few Node.js core modules that still has a built-in polyfill in Webpack 5.

So, when you import the 'events' module in your React app and bundle it with Webpack, Webpack automatically handles the bundling of the 'events' module and includes a polyfill to make it compatible with the browser environment.

To summarize, Webpack is responsible for handling the 'events' polyfill in your project.

**Comment by vincentfretin** [comment]
That's indeed true that webpack 5 removed auto inclusion of polyfills by default.
For the [events](https://www.npmjs.com/package/events) module, I'm not sure we can call it a polyfill in this case. It's just a simple indirect dependency really, webpack depends on events package so that's why you don't have an import error.
